scripts/install-go-tools

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+# Install tools
+#
+
+set -euC
+vimgodir=$(cd -P "$(dirname "$0")/.." > /dev/null && pwd)
+cd "$vimgodir"
+
+vim=$1
+# Make sure all Go tools and other dependencies are installed.
+echo "Installing Go binaries"
+"$vimgodir/scripts/run-vim" $vim +':silent :GoUpdateBinaries' +':qa'
```

Comment:
Oh ... you removed the `export GOPATH=$installdir` :-/

Now it will overwrite people's GOPATHs.